### PR TITLE
Improve BLE_LED instructions.

### PR DESCRIPTION
**Pull request by pan-**
@iriark01 Could you review this ?

### CODE REVIEW

File: BLE_LED/readme.md [hunk 1]
```diff
@@ -34,7 +34,11 @@ Building instructions for all samples are in the [main readme](https://github.co
 1. Build the application and install it on your board as explained in the building instructions.
 1. Open the BLE scanner on your phone.
 1. Find your device.
-1. View the device's characteristics; the exact steps depend on which scanner you're using.
-1. Write the characteristic and see the LED being toggled.
-
-If you can see the characteristic, and the LED is turned on/off as you write to it, the application worked properly.
+1. Find the LED service, its UUID is `0xA000`; the exact steps depend on which scanner you're using.
```
Comment (iriark01) [inline BLE_LED/readme.md]: Find the LED service; its UUID is `0xA000`. The exact instructions to do this depend on the scanner you're using.

File: BLE_LED/readme.md [hunk 2]
```diff
@@ -34,7 +34,11 @@ Building instructions for all samples are in the [main readme](https://github.co
...
+1. Find the LED characteristic, its UUID is `0xA001`; the exact steps depend on which scanner you're using.
```
Comment (iriark01) [inline BLE_LED/readme.md]: Find the LED characteristic; its UUID is 0xA000. The exact instructions to do this depend on the scanner you're using.

File: BLE_LED/readme.md [hunk 3]
```diff
@@ -34,7 +34,11 @@ Building instructions for all samples are in the [main readme](https://github.co
...
+1. The characteristic accept a 1 bytes value:
```
Comment (iriark01) [inline BLE_LED/readme.md]: The characteristic accepts a 1-byte value:

File: BLE_LED/readme.md [hunk 4]
```diff
@@ -34,7 +34,11 @@ Building instructions for all samples are in the [main readme](https://github.co
...
+    * `0x00`: LED ON
+    * `0x01`: LED OFF
+1. Toggle the LED characteristic value and see the LED being toggled.
```
Comment (iriark01) [inline BLE_LED/readme.md]: Don't use "toggle" for both the characteristic and the physical LED.

File: BLE_LED/readme.md [hunk 5]
```diff
@@ -34,7 +34,11 @@ Building instructions for all samples are in the [main readme](https://github.co
...
+
+If you can see the characteristic, and the LED is turned on/off as you toggle its value, the application work properly.
```
Comment (iriark01) [inline BLE_LED/readme.md]: and the LED is turned on and off as you toggle its value, the application is working properly

**Comment by iriark01** [comment]
Left some comments.

File: BLE_LED/readme.md [hunk 6]
```diff
@@ -34,7 +34,11 @@ Building instructions for all samples are in the [main readme](https://github.co
...
-1. View the device's characteristics; the exact steps depend on which scanner you're using.
-1. Write the characteristic and see the LED being toggled.
-
-If you can see the characteristic, and the LED is turned on/off as you write to it, the application worked properly.
+1. Find the LED service; its UUID is `0xA000`. The exact instructions to do this depend on the scanner you're using.
+1. Find the LED characteristic; its UUID is `0xA001`. The exact instructions to do this depend on the scanner you're using.
+1. The characteristic accept a 1-bytes value:
+    * `0x00`: LED ON
+    * `0x01`: LED OFF
+1. Toggle the LED characteristic value and see the LED turn ON or turn OFF, according to the value you set.
```
Comment (pan-) [inline BLE_LED/readme.md]: Irit could you review this sentence ? 
I'm not sure about it. Is the comma needed here ?

**Comment by pan-** [comment]
Thanks, I have updated the PR and added one comment.

File: BLE_LED/readme.md (hunk 6 above)
Comment (iriark01) [inline BLE_LED/readme.md]: Not needed

**Pull request merged**